core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeList.java

```diff
@@ -49,16 +50,10 @@ public SimplifiedMasternodeList(NetworkParameters params) {
     SimplifiedMasternodeList(SimplifiedMasternodeList other, short version) {
         super(other.params);
         this.version = version;
-        this.protocolVersion = version == SimplifiedMasternodeListDiff.BASIC_BLS_VERSION ?
-                NetworkParameters.ProtocolVersion.CURRENT.getBitcoinProtocolVersion() :
-                NetworkParameters.ProtocolVersion.BLS_LEGACY.getBitcoinProtocolVersion();
+        this.protocolVersion = params.getProtocolVersionNum(NetworkParameters.ProtocolVersion.SMNLE_VERSIONED);
```

Comment:
when creating a new `SimplifiedMasternodeList`, we will the most recent known protocol version that will be used to save data.